integration/pmem/pmem_test.sh

```diff
@@ -42,14 +42,65 @@ function setup() {
 	fi
 }
 
-function test_pmem {
+function test_pmem_mount {
 	"${dir_path}/../../cmd/pmemctl/pmemctl.sh" -s 128M -f xfs -m "${test_directory}" xfs.img
 
 	# Running container
-	docker run -d --name "${container_name}" --runtime kata-runtime -v "${test_directory}:/${test_directory_name}" "${image}" sh -c "${payload}"
+	docker run -d --name "${container_name}" --runtime ${RUNTIME} -v "${test_directory}:/${test_directory_name}" "${image}" sh -c "${payload}"
 
 	# Check container
 	docker exec "${container_name}" sh -exc "mount | grep ${test_directory_name} | grep '/dev/pmem' | grep 'dax'"
+
+	sudo umount "${test_directory}"
+	sudo losetup -D
+	rm -f xfs.img
+}
+
+function wait_for_postgres {
+	container="$1"
+
+	for i in $(seq 1 20); do
+		if docker exec -u postgres "${container}" psql -c '\l'; then
+			break
+		fi
+		sleep 3
+	done
+}
+
+function test_database {
+	"${dir_path}/../../cmd/pmemctl/pmemctl.sh" -s 1G -f xfs -m "${test_directory}" xfs.img
+
+	rows=1000000
+	cont_image=postgres
+	postgresql_cont_dir="/var/lib/postgresql/data"
```

Comment:
How about creating a `integration/pmem/data/lib.sh` for this variable, the db name, db password, etc that could be sourced by these two scripts?